PR failing on Jenkins with "org.eclipse.jdt.core.tests.builder.mockcompiler: No tests found"

Regression from https://github.com/eclipse-jdt/eclipse.jdt.core/pull/5012.

See https://github.com/eclipse-jdt/eclipse.jdt.core/pull/5012#issuecomment-4280280559

Currently all PR's for JDT core based on latest master fail with the error below
```
[ERROR] Failed to execute goal org.eclipse.tycho:tycho-surefire-plugin:5.0.3-SNAPSHOT:test (default-test) on project org.eclipse.jdt.core.tests.builder.mockcompiler: No tests found -> [Help 1]
14:15:52  org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.eclipse.tycho:tycho-surefire-plugin:5.0.3-SNAPSHOT:test (default-test) on project org.eclipse.jdt.core.tests.builder.mockcompiler: No tests found
14:15:52      at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:333)
14:15:52      at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
```

--- a/org.eclipse.jdt.core.tests.builder.mockcompiler/pom.xml
+++ b/org.eclipse.jdt.core.tests.builder.mockcompiler/pom.xml
@@ -21,5 +21,10 @@
   </parent>
   <artifactId>org.eclipse.jdt.core.tests.builder.mockcompiler</artifactId>
   <version>1.0.200-SNAPSHOT</version>
+  <!-- Workaround for API baselines tycho problem: don't add this bundle to baselines check -->
   <packaging>eclipse-test-plugin</packaging>
+  <properties>
+    <!-- This plugin is used by tests but has no tests itself, see packaging comment above -->
+    <failIfNoTests>false</failIfNoTests>
+  </properties>
 </project>

